rportfolio/src/index.js

```diff
@@ -2,7 +2,6 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
```

Comment:
Do you still have this file? I can't find it. It may cause unexpected errors, please remove this line